### ISSUE TITLE
`Unhandled exception. System.InvalidOperationException: Sequence contains no matching element`

### ISSUE THREAD

**Issue by nigeljmckernan** [MrPnut/SCXI#3]
Out the gate, when trying to open `SCXI` I get the error in the issue title.

Corresponding screenshot.

Tried with Steam Controller connected and disconnected, tried with Steam open and Steam closed. Nothing works.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/71948964/167215245-755693b8-b8ab-46c2-b1fe-76d6fda3150a.png">

`.NET` version 5 and 6 runtimes are installed, along with the latest version of `ViGEmBus`.

**Comment by MrPnut** [comment]
@nigeljmckernan 

Interesting.  Are you using the controller wired by any chance?  I've actually only tested it with the wireless dongle, so that might be what it is.

**Comment by nigeljmckernan** [comment]
Nope. I exclusively use my Steam Controller via Bluetooth to my PC.

**Comment by MrPnut** [comment]
Hmm.. 

I've only used the official wireless receiver and I've not used the bluetooth functionality.  I'll mess around with trying to get it to work with bluetooth and let you know.

Thanks for opening the issue!

**Comment by MrPnut** [comment]
I still am looking into this.  When using BT, there are a few HID devices that are present on the system.  Unfortunately, if I try to read them with CreateFile, I get access denied.

I'll keep this ticket updated as I continue to research and debug.
